homeassistant/components/decora_ble/config_flow.py

```diff
@@ -0,0 +1,155 @@
+"""Config flow for Decora BLE."""
+from __future__ import annotations
+
+import logging
+from typing import Any, Union
+
+from decora_bleak import DECORA_SERVICE_UUID, DecoraBLEDevice
+import voluptuous as vol
+
+from homeassistant.components import bluetooth
+from homeassistant.components.bluetooth import (
+    BluetoothServiceInfoBleak,
+    async_discovered_service_info,
+)
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_ADDRESS, CONF_API_KEY, CONF_NAME
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+_LOGGER.setLevel(logging.DEBUG)
+
+DEFAULT_NAME = "Room Lights"
```

Comment:
It would be better to store nothing instead of a default here so we know in the future if its unset in the event we can somehow get the name for the device and use it when the user didn't set it before.